metric-schema-java/src/main/java/com/palantir/metric/schema/UtilityGenerator.java

```diff
@@ -50,7 +52,7 @@ static JavaFile generateUtilityClass(
                 .addModifiers(visibility.apply(Modifier.FINAL))
                 .addJavadoc(Javadoc.render(metrics.getDocs()))
                 .addMethod(MethodSpec.methodBuilder(ReservedNames.FACTORY_METHOD)
-                        .addModifiers(Modifier.PUBLIC, Modifier.STATIC)
+                        .addModifiers(visibility.apply(Modifier.STATIC))
```

Comment:
replaced a couple places which specified the public modifier despite being unnecessary based on the encapsulating classes visibility (no consumer impact, just cleaner generated code using a utility we already have)